### PR TITLE
Bear.py: Maintainers are authors by default

### PR DESCRIPTION
**Pull request by Adrianzatreanu**

### CODE REVIEW

**Comment by Asnelchristian** [comment]
ack https://github.com/coala-analyzer/coala/pull/2352/commits/6f7b50e488e02b3e6d57a793f65bc89087438957

File: coalib/bears/Bear.py [hunk 1]
```diff
@@ -113,8 +113,8 @@ class Bear(Printer, LogPrinter):
     REQUIREMENTS = ()
     AUTHORS = set()
     AUTHORS_EMAILS = set()
-    MAINTAINERS = set()
-    MAINTAINERS_EMAILS = set()
+    MAINTAINERS = AUTHORS
+    MAINTAINERS_EMAILS = AUTHORS_EMAILS
```
Comment (Adrianzatreanu) [inline coalib/bears/Bear.py]: @sils1297 take the emails as well, right?

Comment (sils1297) [inline coalib/bears/Bear.py]: yes, also this code won't work, rght?

Comment (Adrianzatreanu) [inline coalib/bears/Bear.py]: im not sure, thats why i did not merge it yet, i asked on gitter and you said its enough to do this :p
so should i make it a class property or what?

Comment (Adrianzatreanu) [inline coalib/bears/Bear.py]: something like: if maintainers is ``set()``, maintainers = authors?

File: coalib/bears/Bear.py [hunk 2]
```diff
@@ -74,11 +74,20 @@ class Bear(Printer, LogPrinter):
 
     >>> class SomeBear(Bear):
     ...     AUTHORS = {'Jon Snow'}
-    ...     AUTHORS_EMAIL = {'jon_snow@gmail.com'}
+    ...     AUTHORS_EMAILS = {'jon_snow@gmail.com'}
     ...     MAINTAINERS = {'Catelyn Stark'}
     ...     MAINTAINERS_EMAILS = {'catelyn_stark@gmail.com'}
     ...     LICENSE = 'AGPL-3.0'
 
+    If the maintainers are the same as the authors, they can be omitted:
```
Comment (sils1297) [inline coalib/bears/Bear.py]: add newline here

File: coalib/bears/Bear.py [hunk 3]
```diff
@@ -136,6 +146,19 @@ def can_detect(cls):
         """
         return cls.CAN_DETECT | cls.CAN_FIX
 
+    @classproperty
+    def maintainers(cls):
+        """
```
Comment (Adrianzatreanu) [inline coalib/bears/Bear.py]: ah nvm i noticed i forgot to put those just in rush to see if its working, will fix this in 20-30mins when i got time again

**Comment by Adrianzatreanu** [comment]
Due to problems with git and accidentally corrupting files, moved new PR here https://github.com/coala-analyzer/coala/pull/2353

**Pull request closed without merging**